easybuild/easyconfigs/f/freebayes/freebayes-1.1.0-goolf-1.7.20_39e5e4b.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'MakeCp'
+
+name = 'freebayes'
+version = '1.1.0'
+versionsuffix = '_39e5e4b'
+
+homepage = 'https://github.com/ekg/freebayes'
+description = """ Bayesian haplotype-based polymorphism discovery and genotyping. """
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no direct download url. As described in documentation the download has been done with
+# $> git clone --recursive git://github.com/ekg/freebayes.git
+# Then compressed and placed in the sources folder with name "freebayes-1.1.0_39e5e4b.tar.gz"
+sources = ['%(name)s-%(version)s%(versionsuffix)s.tar.gz']
```

Comment:
@pescobar what's wrong with https://github.com/ekg/freebayes/archive/v1.1.0.tar.gz ?

according to https://github.com/ekg/freebayes/releases, that's exactly the same as commit ` 39e5e4b`...

so, use this?

```python
source_urls = ['https://github.com/ekg/freebayes/archive/']
sources = ['v%(version)s.tar.gz']
```

and then drop the `versionsuffix`?